### PR TITLE
Update rem mixin to return 1px for rem(1px)

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/frontend Please review

**Comment by perry** [comment]
@wytrych please can you update the [changelog.md](https://github.com/incuna/incuna-sass/blob/master/CHANGELOG.md) and [bower.json](https://github.com/incuna/incuna-sass/blob/master/bower.json), I see this as a bug fix, so version `4.0.4` will be fine.

File: incuna-sass/functions/_rem.sass [hunk 1]
```diff
@@ -5,6 +5,8 @@ $use-rem: false !default
     @if $use-rem
         @if $target == 0
             @return 0
+        @if $target == 1px
+            @return 1px
```
Comment (perry) [inline incuna-sass/functions/_rem.sass]: IIRC, `2px` was also a problem? Perhaps we could check `$target` for these values on line 5, returning `$target` without modification on line 12 if it's in the blacklist?

Comment (grahamgilchrist) [inline incuna-sass/functions/_rem.sass]: We haven't seen any problems using `2px` recently. Lets add that in later if it becomes an issue.

**Comment by grahamgilchrist** [comment]
@wytrych Now you just need to tag the repo as per the release instructions in the readme. Then you can reference this in other projects by version tag.

File: incuna-sass/functions/_rem.sass (hunk 1 above)
Comment (henrahmagix) [inline incuna-sass/functions/_rem.sass]: `2px` would only be a problem with a `$font-base` much below `8px`, which is unlikely to happen.

**Comment by henrahmagix** [comment]
:+1: nice update

**Pull request merged**